pkg/asset/cluster/tfvars.go

```diff
@@ -122,8 +124,12 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
-		data, err = libvirttfvars.TFVars(
-			masters[0].Spec.ProviderSpec.Value.Object.(*libvirtprovider.LibvirtMachineProviderConfig),
+		masterConfigs := make([]*libvirtprovider.LibvirtMachineProviderConfig, len(masters))
+		for i, m := range masters {
+			masterConfigs[i] = m.Spec.ProviderSpec.Value.Object.(*libvirtprovider.LibvirtMachineProviderConfig)
+		}
+		data, err := libvirttfvars.TFVars(
+			masterConfigs,
```

Comment:
I don't see a point to passing multiple provider specs through to the libvirt or OpenStack Terraform generators until we have code consuming them.  We may have that code at some point though; are we confident enough to commit to it now?

Reply:
The only point is consistency. Let the platform implementation decide what they are going to do with the provider configs rather than have that decision made by the Terraform Variables asset.

Reply:
I've removed the controversial changes.